src/main/java/ladder/domain/LadderLine.java

```diff
@@ -0,0 +1,49 @@
+package ladder.domain;
+
+import java.util.List;
+
+public class LadderLine {
+    private final List<Boolean> points;
+
+    public LadderLine(List<Boolean> points) {
+        this.points = points;
+    }
+
+    public int move(int startPosition) {
+        if (startPosition < 0 || startPosition > points.size()) {
+            throw new IllegalArgumentException("Invalid start position");
+        }
+
+        if (isRightConnected(startPosition)) return startPosition + 1;
+        if (isLeftConnected(startPosition)) return startPosition - 1;
+
+        return startPosition;
+    }
+
+    public int size() {
+        return points.size();
+    }
+
+    public boolean getPoint(int index) {
+        if (index < 0 || index >= points.size()) {
+            throw new IllegalArgumentException("Invalid index");
+        }
+        return points.get(index);
+    }
+
+    private boolean isLeftConnected(int startPosition) {
+        if (startPosition == 0) {
+            return false;
+        }
+
+        return points.get(startPosition - 1);
```

Comment:
```suggestion
        return position > 0 && points.get(position - 1);
```
이런식으로 작성하는것도 동일하지 않을까요?